tests/unit/components/SpellingVariantInput.test.ts

```diff
@@ -45,15 +45,72 @@ describe( 'SpellingVariantInput', () => {
 			expect( wikitLookup.props( 'menuItems' ) ).toStrictEqual( [
 				{
 					description: '',
-					label: 'en',
+					label: 'English (en)',
+					value: 'en',
 				},
 				{
 					description: '',
-					label: 'en-ca',
+					label: 'British English (en-gb)',
+					value: 'en-gb',
 				},
 			] );
 		} );
 	} );
+
+	describe( 'suggested options', () => {
+		it.each( [
+			[
+				'Tun would open Tunisia',
+				[ [ 'en', 'English' ], [ 'aeb-latn', 'Tunisian Arabic (Latin script)' ], [ 'de', 'German' ] ],
+				'Tun',
+				[ 'aeb-latn' ],
+			],
+			[
+				'matching is not fuzzy - tsa does not match Tunisia',
+				[ [ 'en', 'English' ], [ 'aeb-latn', 'Tunisian Arabic (Latin script)' ], [ 'de', 'German' ] ],
+				'tsa',
+				[],
+			],
+			[
+				'Matching looks at each word in the list, i.e. Tunisian Arabic would display if arabic entered',
+				[ [ 'en', 'English' ], [ 'aeb-latn', 'Tunisian Arabic (Latin script)' ], [ 'de', 'German' ] ],
+				'arabic',
+				[ 'aeb-latn' ],
+			],
+			[
+				'Matching looks at each word in the list, i.e. Tunisian Arabic would display if arabic entered',
```

Comment:
I assume this is supposed to be a test for looking at the language code?

Reply:
ah yes. I'll fix the message

Reply:
Done